en_US/changes/changes-ee-5.1.md

```diff
@@ -1,5 +1,233 @@
+
+- [#11151](https://github.com/emqx/emqx/pull/11151) The MySQL bridge has been refactored to its own Erlang application to improve the code structure and to make it easier to maintain.
+
+- [#11163](https://github.com/emqx/emqx/pull/11163) Hid `topology.pool_size` in MondoDB bridges and fixed it to 1 to avoid confusion.
+
+- [#11175](https://github.com/emqx/emqx/pull/11175) Now when using a nonexistent hostname for connecting to MySQL, a 400 error is returned rather than 503 in the REST API.
+
+- [#11198](https://github.com/emqx/emqx/pull/11198) Fixed global rebalance status evaluation on replicant nodes. Previously, `/api/v5/load_rebalance/global_status` API method could return incomplete results if handled by a replicant node.
+  
+- [#11223](https://github.com/emqx/emqx/pull/11223) In InfluxDB bridging, mixing decimals and integers in a field may lead to serialization failure in the Influx Line Protocol, resulting in the inability to write to the InfluxDB bridge (when the decimal point is 0, InfluxDB mistakenly interprets it as an integer)..
+
+  See also: [InfluxDB v2.7 Line-Protocol](https://docs.influxdata.com/influxdb/v2.7/reference/syntax/line-protocol/#float).
+
+- [#11225](https://github.com/emqx/emqx/pull/11225) The `username` field in PostgreSQL/Timescale/MatrixDB bridges configuration is now a required one.
+
+- [#11242](https://github.com/emqx/emqx/pull/11242) Restarted emqx_ee_schema_registry when a node joins a cluster. As emqx_ee_schema_registry uses Mria tables, a node joining a cluster needs to restart this application in order to start relevant Mria shard processes, ensuring a correct behaviour in Core/Replicant mode.
+
+- [#11266](https://github.com/emqx/emqx/pull/11266) Fixed and improved support for TDengine `insert` syntax:
+
+  1. Added support for inserting into multi-table in the template.
+
+     For example:
+
+     `insert into table_1 values (${ts}, '${id}', '${topic}')
+     table_2 values (${ts}, '${id}', '${topic}')`
+
+  2. Added support for mixing prefixes/suffixes and placeholders in the template.
+
+     For example:
+
+     `insert into table_${topic} values (${ts}, '${id}', '${topic}')`
+
+     Note: This is a breaking change. Previously, the values of string type were quoted automatically, but now they must be quoted explicitly.
+
+     For example:
+
+     `insert into table values (${ts}, '${a_string}')`
+
+- [#11307](https://github.com/emqx/emqx/pull/11307) Fixed check for table existence to return a more friendly message in the Oracle bridge.
+
+- [#11316](https://github.com/emqx/emqx/pull/11316) Fixed Pool Size value not being considered in Oracle Bridge.
+
+- [#11326](https://github.com/emqx/emqx/pull/11326) Fixed return error checking on table validation in the Oracle bridge.
+
+### [Known Issues](https://github.com/emqx/emqx-docs/blob/release-5.1/en_US/changes/known-issues-5.1.1.md)
```

Comment:
this document does not exist

Reply:
This file will be added after sorting out "Known Issues"